src/Kernel/Magic/EstimateMagic.cs

```diff
@@ -28,9 +28,40 @@ public class EstimateMagic : AbstractMagic
         /// </summary>
         public EstimateMagic(ISymbolResolver resolver) : base(
             "estimate",
-            new Documentation
-            {
-                Summary = "Runs a given function or operation on the ResourcesEstimator target machine."
+            new Documentation {
+                Summary = "Runs a given function or operation on the ResourcesEstimator target machine.",
+                Description = @"
```

Comment:
Would a link to the resources estimator docs be helpful somewhere in this Description?

Reply:
Yes. Similarly with the QuantumSimulator and ToffoliSimulator, I think. Good idea.